DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/task_sequences/README.md

```diff
@@ -55,70 +55,109 @@ You can also find more information about [troubleshooting build errors](/DirectP
 ## Key Implementation Details
 
 This sample demonstrates some key concepts:
-- Basics of instantiating objects of parameterized instance of task_sequence.
-- Using async() function to invoke task functions for the task_sequence object.
+- How to execute multiple dependent loops in parallel using the `task_sequence` class, `pipe`s and the `async()` function call
+- How the compiler automatically adds depth to pipes that bypass one or more loops
 
-This design demonstrates a system structured like this:
+The demonstration system in this tutorial is structured like this:
 
 ![](assets/LoopStructure.svg)
 
-When you write code containing a sequence of loops, each loop in the sequence must wait for the previous loop in the sequence to fully complete. This results in long execution time and poor occupancy of the FPGA datapath:
+When you write code containing a sequence of loops, each loop in the sequence must wait for the previous loop in the sequence to fully complete. This results in long execution time and poor occupancy of the FPGA datapath. Observe in the following graphic that it takes many hundreds of clock cycles before the output pipe produces any output:
 
 ![](assets/simulation_naive.png)
 
-Observe in the above graphic that it takes many hundreds of clock cycles before the output pipe produces any output.
-Task sequences allow you to schedule loops to run concurrently, so data quickly flows through sequences of loops.
+Task sequences allow you to schedule loops to run in parallel, so data quickly flows through sequences of loops. A consumer loop can process an output as soon as a producer loop produces it, without waiting for the whole producer loop to finish executing. Observe in the following graphic that the output pipe produces results almost immediately:
 
 ![](assets/simulation_task_sequence.png)
 
-Observe in the above graphic that the output pipe produces results almost immediately.
 
```

Comment:
Add a note that this is only possible when the loops process data in the same order. That is, the **first** piece of data the second loop consumes should be the **first** piece of data produced by the first loop. Converting loops to task sequences and executing them in parallel will not help performance if the **first** piece of data consumed by the second loop is the **last** piece of data produced by the first loop, because then the second loop requires the first loop to completely finish.


Feel free to make that shorter if you like :)